src/shared/menu/entries.ts

```diff
@@ -335,9 +335,12 @@ export const createMenuEntries = (
                 commonmark: inlineCodeCommand,
                 display: makeMenuButton(
                     "Code",
-                    _t("commands.inline_code", {
-                        shortcut: getShortcut("Mod-K"),
-                    }),
+                    {
```

Comment:
@b-kelly I understand your good intentions here but we cannot take this kind of executive decisions after a group of designers have put an effort to analyse the story. I might even agree with you here but we need to make these suggestions transparent.